packages/components/Tabs/src/TabsItem.tsx

```diff
@@ -43,6 +43,9 @@ export const TabsItem = compose<TabsItemType>({
     We check that there is an actual change in selection before forwarding the message to the user callback
     so that they aren't notified more than once for each click. */
     const changeSelection = () => {
+      if (Platform.OS === 'windows') {
+        info.focusZoneRef.current.focus();
+      }
```

Comment:
I thought there is no FocusZone on windows so the ref would be null?

Reply:
It is the reference to the View (with JS keyboarding) that acts as a "focusZone". Should I change the name of the variable to something more clear?

Reply:
What is the reference in the case of windows? Like, which slot? 
It kinda feels like we should just take the JS code you wrote to mimic a FocusZone, and put it in the FocusZone component as the windows implementation of FocusZone, and keep the logic inside Tabs simpler. I'm not sure how complicated of a change that is to implement though, especially since you all will finish your internship soon.

Reply:
It is a reference to the stack slot.

Reply:
It might be a bit complicated since the JS code that mimics the FocusZone is ingrained to the Tabs Control. But it is an interesting proposition to consider.

Reply:
We talked offline, and it sounds like we're not trying to make a JS FocusZone for windows with this review. I would leave a comment with the GH issue number for this piece of code as well. Sorry for being pedantic, I tend to recommend over-documenting rather than under-documenting.